packages/remix-forms/src/createField.tsx

```diff
@@ -279,6 +291,21 @@ function createField<Schema extends SomeZodObject>({
       ref,
     ) => {
       const value = fieldType === 'date' ? parseDate(rawValue) : rawValue
+      const field = {
```

Comment:
it would be great to reuse this constant to pass the `childrenFn` parameters (line 320) for the sake of readability, it makes it clearer that we are talking about the same `field` object.